File: app/src/main/java/com/ikvych/cocktail/util/ActivityUtil.java
package com.ikvych.cocktail.util;

import android.app.Activity;
import android.view.View;

import com.ikvych.cocktail.R;

public class ActivityUtil {

    public static void setSearchEmptyListVisible(Activity activity) {
        activity.findViewById(R.id.empty_list_id).setVisibility(View.VISIBLE);
        activity.findViewById(R.id.empty_search_id).setVisibility(View.GONE);
        activity.findViewById(R.id.search_recycler_view).setVisibility(View.GONE);
    }

    public static void setSearchRecyclerViewVisible(Activity activity) {
        activity.findViewById(R.id.empty_list_id).setVisibility(View.GONE);
        activity.findViewById(R.id.empty_search_id).setVisibility(View.GONE);
        activity.findViewById(R.id.search_recycler_view).setVisibility(View.VISIBLE);
    }

    public static void setEmptySearchVisible(Activity activity) {
        activity.findViewById(R.id.empty_list_id).setVisibility(View.GONE);
        activity.findViewById(R.id.empty_search_id).setVisibility(View.VISIBLE);
        activity.findViewById(R.id.search_recycler_view).setVisibility(View.GONE);
    }

    public static void setDbRecyclerViewVisible(Activity activity) {
        activity.findViewById(R.id.empty_history).setVisibility(View.GONE);
        activity.findViewById(R.id.db_recycler_view).setVisibility(View.VISIBLE);
    }

    public static void setDbEmptyHistoryVisible(Activity activity) {
        activity.findViewById(R.id.empty_history).setVisibility(View.VISIBLE);
        activity.findViewById(R.id.db_recycler_view).setVisibility(View.GONE);
    }
}

File: app/src/main/java/com/ikvych/cocktail/util/ActivityUtil.kt
package com.ikvych.cocktail.util

import android.app.Activity
import android.view.View
import com.ikvych.cocktail.R

fun setSearchEmptyListVisible(activity: Activity) {
    activity.findViewById<View>(R.id.tv_begin_search).visibility = View.VISIBLE
    activity.findViewById<View>(R.id.tv_empty_search_result).visibility = View.GONE
    activity.findViewById<View>(R.id.rv_search_result).visibility = View.GONE
}

fun setSearchRecyclerViewVisible(activity: Activity) {
    activity.findViewById<View>(R.id.tv_begin_search).visibility = View.GONE
    activity.findViewById<View>(R.id.tv_empty_search_result).visibility = View.GONE
    activity.findViewById<View>(R.id.rv_search_result).visibility = View.VISIBLE
}

fun setEmptySearchVisible(activity: Activity) {
    activity.findViewById<View>(R.id.tv_begin_search).visibility = View.GONE
    activity.findViewById<View>(R.id.tv_empty_search_result).visibility = View.VISIBLE
    activity.findViewById<View>(R.id.rv_search_result).visibility = View.GONE
}

fun setDbRecyclerViewVisible(view: View) {
    view.findViewById<View>(R.id.tv_empty_history).visibility = View.GONE
    view.findViewById<View>(R.id.rv_search_result).visibility = View.VISIBLE
}

fun setDbEmptyHistoryVisible(view: View) {
    view.findViewById<View>(R.id.tv_empty_history).visibility = View.VISIBLE
    view.findViewById<View>(R.id.rv_search_result).visibility = View.GONE
}
